content/_code-samples/checks/js/get_checks.js

```diff
@@ -0,0 +1,56 @@
+'use strict'
+const xrpl = require('xrpl');
+
+const address = "rEYyvXCwv1WJ6a8hW4sBx5afXbJTK97bbR"; // <-- Change this to your account
+
+const main = async () => {
+  try {
+    // Connect ----------------------------------------------------------------
+    const client = new xrpl.Client('wss://s.altnet.rippletest.net:51233');
+    await client.connect();
+
+    let currMarker = null;
+    let account_objects = [];
+
+    // Loop through all account objects until marker is undefined --------------
+    let keepLooping = true;
+    
+    while(keepLooping){
```

Comment:
You can probably replace `keepLooping` with a `do...while(marker === undefined)` loop that stops when marker stops existing.

Reply:
I am questioning myself of why I didn't think of that, haha. Done!